### ISSUE TITLE
Add command to insert ignore line for linting errors

### ISSUE THREAD

**Issue by entropitor** [dense-analysis/ale#3348]
Some linting rules indicate potential errors and thus it's not uncommon to have to add e.g. "eslint-disable-next-line". It would be cool if there was a command you could run which would insert this for you based on actual errors. 

E.g. see `eslint.codeAction.disableRuleComment` from https://github.com/Microsoft/vscode-eslint

**Comment by w0rp** [comment]
Had this idea myself previously, and I want to do it, especially for `shellcheck`.

**Comment by svengit** [comment]
Yes, I really need this. I am currently evaluating if we can switch from Syntastic to ALE and all `# shellcheck disable=` lines get ignored which is a dealbreaker.
